### PR TITLE
[zwave-js-ui] Add support for subPath to persistence

### PR DESCRIPTION
**Pull request by gandazgul**
Added support for specifying a subPath in persistence. I used a shared volume for all configurations and each one is on a subPath from root.

### CODE REVIEW

File: charts/zwave-js-ui/templates/deployment.yaml [hunk 1]
```diff
@@ -82,7 +82,13 @@ spec:
           volumeMounts:
             - name: store
               mountPath: {{ .Values.persistence.mountPath }}
+              subPath: {{ .Values.persistence.subPath }}
       volumes:
+        {{- if .Values.usbDevice }}
```
Comment (varet80) [inline charts/zwave-js-ui/templates/deployment.yaml:91]: can you also Add usbDevice in values.yaml?

**Comment by varet80** [comment]
@gandazgul thank you for this contribution. added a small comment, for the documentation.
Did you bump the version of the chart, or was it automatically done?

**Comment by varet80** [comment]
I am on vacation, and release action broke.
Once I fix it, I will merge your PR

**Comment by gandazgul** [comment]
thank you!

**Pull request merged**